src/components/sandboxLink.tsx

```diff
@@ -0,0 +1,86 @@
+import React from "react";
+import usePlatform from "./hooks/usePlatform";
+
+
+export enum Scenario {
```

Comment:
@Neo-Zhixing why export this?

Reply:
why not?

Reply:
@Neo-Zhixing I don't see why we'd need to use this anywhere else

Reply:
If you're calling `getSandboxURL` you'd need to have `Scenario`

Reply:
@Neo-Zhixing I see, I think this might be because of how you are typing it. If you typed it like in my other [comment](https://github.com/getsentry/sentry-docs/pull/4354/files#r742279479), then you can just pass in `scenario` as literal string like `discover` and then you wouldn't need to import this type.